opencensus/stats/internal/aggregation.cc

```diff
@@ -33,6 +33,8 @@ std::string Aggregation::DebugString() const {
       return absl::StrCat("Distribution with ",
                           bucket_boundaries_.DebugString());
     }
+    default:
+      return std::string();
```

Comment:
Please move this after the switch() statement. The reason is that providing a default: case suppresses warning about missing a case in an enum.

Reply:
Done